packages/common/src/fixtures/preview-service/xforms/all-question-types-v2024091201.xml

```diff
@@ -1,4 +1,4 @@
_question_types/myanmar_date" type="date"/><bind nodeset="/data/date_time_question_types/persian_date" type="date"/><bind nodeset="/data/date_time_question_types/buddhist_date" type="date"/><bind nodeset="/data/geo_question_types/geopoint" type="geopoint"/><bind nodeset="/data/geo_question_types/geopoint_placementmap" type="geopoint"/><bind nodeset="/data/geo_question_types/geopoint_maps" type="geopoint"/><bind nodeset="/data/geo_question_types/geotrace" type="geotrace"/><bind nodeset="/data/geo_question_types/geoshape" type="geoshape"/><bind nodeset="/data/select_one_question_types/select_one" type="string"/><bind nodeset="/data/select_one_question_types/select_one_minimal" type="string"/><bind nodeset="/data/select_one_question_types/select_one_autoadvance" type="string"/><bind nodeset="/data/select_one_question_types/select_one_autocomplete" type="string"/><bind nodeset="/data/select_one_question_types/grid_columns_pack" type="string"/><bind nodeset="/data/select_one_question_types/grid_columns" type="string"/><bind nodeset="/data/select_one_question_types/grid_columns2" type="string"/><bind nodeset="/data/select_one_question_types/grid_columns_no_buttons" type="string"/><bind nodeset="/data/select_one_question_types/select_one_image_map" type="string"/><bind nodeset="/data/select_one_question_types/likert" type="string"/><bind nodeset="/data/select_one_question_types/map" type="string"/><bind nodeset="/data/select_one_question_types/select_one_final_note" readonly="true()" type="string"/><bind nodeset="/data/select_multi_question_types/select_multi" type="string"/><bind nodeset="/data/select_multi_question_types/select_multi_autocomplete" type="string"/><bind nodeset="/data/select_multi_question_types/grid_multi_columns_pack" type="string"/><bind nodeset="/data/select_multi_question_types/grid_multi_columns" type="string"/><bind nodeset="/data/select_multi_question_types/grid_multi_columns2" type="string"/><bind nodeset="/data/select_multi_question_types/grid_multi_columns_no_buttons" type="string"/><bind nodeset="/data/select_multi_question_types/select_multi_minimal" type="string"/><bind nodeset="/data/select_multi_question_types/select_multi_image_map" type="string"/><bind nodeset="/data/select_multi_question_types/select_multiple_final_note" readonly="true()" type="string"/><bind nodeset="/data/table_list/table_list_label" type="string"/><bind nodeset="/data/table_list/table_list_label_2" type="string"/><bind nodeset="/data/table_list/table_list_1" type="string"/><bind nodeset="/data/table_list/table_list_2" type="string"/><bind nodeset="/data/table_list/list" type="string"/><bind nodeset="/data/table_list/list_multi" type="string"/><bind nodeset="/data/other_question_types/rank" type="odk:rank"/><bind nodeset="/data/meta/instanceID" type="string" readonly="true()" jr:preload="uid"/></model></h:head><h:body><input ref="/data/intro"><label>Welcome to ODK! This form showcases the different available question types, sometimes called widgets.
```

Comment:
This is a standard form maintained at https://docs.google.com/spreadsheets/d/1af_Sl8A_L8_EULbhRLHVl8OclCfco09Hq2tqb9CslwQ/edit?gid=0#gid=0 and with the version string in the filename. I think the change here is to the choice options. It seems reasonable to change them in the source form because they're more illustrative than the current a, b, c, d icons. I can make the change to the source XLSForm then you can include the updated one and change the filename?

Reply:
As far as I know, png and jpg images are much more common than svg. Should we maybe include some of those as well?

Reply:
I see! Yes, please 

I changed the labels because single letters weren't helpful in detecting problems with the display of labels and responsive columns. I included mostly SVGs because they are lighter than PNG (few KBs).

It already has a couple of PNGs and SVGs. I'm going to change a couple of SVGs to JPG, so it has the 3 image types.

Reply:
Done, waiting for the new form to update the filenames